main/src/ca/uwaterloo/flix/language/phase/Parser2.scala

```diff
@@ -783,13 +806,13 @@ object Parser2 {
     assert(at(TokenKind.KeywordUse))
     val mark = open()
     expect(TokenKind.KeywordUse, SyntacticContext.Use)
-    name(NAME_USE, allowQualified = true, context = SyntacticContext.Use)
+    nameAllowQualified(NAME_USE, context = SyntacticContext.Use)
     // handle use many case
     if (at(TokenKind.DotCurlyL)) {
       val mark = open()
       oneOrMore(
         namedTokenSet = NamedTokenSet.Name,
-        getItem = () => aliasedName(NAME_USE, tail = Set(TokenKind.NameLowerCase), SyntacticContext.Use),
+        getItem = () => aliasedName(NAME_USE, SyntacticContext.Use),
```

Comment:
Here was an example of a given tail set that is unused because allowQualified was false - just as a example of why this is needed